### DISCUSSION TITLE
Issue connecting to Azure sql server

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#5012] [Q&A] [answered]
TLS version 1 was disabled (this past weekend) on the azure sql server database  that my channel was connecting to. Now it fails with this error:

Error: JavaException: java.sql.SQLException: Reason: Login failed due to client TLS version being less than minimal TLS version allowed by the server.

I've tried these two drivers with no luck

dbConn = DatabaseConnectionFactory.createDatabaseConnection('net.sourceforge.jtds.jdbc.Driver',dbURL,dbUserName,dbPassword);
dbConn = DatabaseConnectionFactory.createDatabaseConnection('com.microsoft.sqlserver.jdbc.SQLServerDriver',dbURL,dbUserName,dbPassword);

Anyone else have this issue or a workaround?

**Comment by rutledj** [comment, marked as answer]
This fixed the problem, should anyone else run across this issue.

https://stackoverflow.com/questions/67310070/azure-sql-is-there-anyway-to-connect-to-azure-sql-server-using-jtds-with-tls-1

**Comment by jonbartels** [comment]
Would an alternative solution be to use the MS SQL JDBC driver instead of JTDS?

**Comment by rutledj** [comment]
I tried both and neither worked until adding the above.
